config/newrelic.yml

```diff
@@ -16,7 +20,9 @@
     explain_threshold: 0.5
   error_collector:
     enabled: true
-    capture_source: true
```

Comment:
`capture_source` is set to `true` by default.

Reply:
Looks like that setting has been removed entirely: https://docs.newrelic.com/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration/

Reply:
This setting is still in the full New Relic config; it's just not documented. There are a lot more settings in the full config than what's on that page.

Reply:
Actually, I may be wrong here. I'll do some additional testing to confirm whether this still exists.

Reply:
Confirmed that this setting has been removed entirely from the gem. It was persisting in the New Relic UI when I tested it yesterday, but according to the git log for the gem, this setting was removed in 2014.